meteo-notebook.ipynb

```diff
@@ -110,10 +110,27 @@
   },
```

Comment:
<p>I think it would be clearer to write the entire date.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Changed it to entire date